plugins/source/aws/resources/services/lightsail/instance_port_states.go

```diff
@@ -17,14 +17,17 @@ func instancePortStates() *schema.Table {
 		Name:        tableName,
 		Description: `https://docs.aws.amazon.com/lightsail/2016-11-28/api-reference/API_InstancePortState.html`,
 		Resolver:    fetchLightsailInstancePortStates,
-		Transform:   transformers.TransformWithStruct(&types.InstancePortState{}),
+		Transform: transformers.TransformWithStruct(&types.InstancePortState{},
+			transformers.WithPrimaryKeys("FromPort", "ToPort", "Protocol"),
```

Comment:
access to specific ports is based on source addresses so I think we need to include `cidr` and `ipv6cidr` attributes as well

Reply:
4e4858978bf4812ce1f47f41da45907545a26112
inspired by [`aws cli lightsail put-instance-public-ports`](https://docs.aws.amazon.com/cli/latest/reference/lightsail/put-instance-public-ports.html) doc